src/pages/home/sidebar/SidebarScreen/FloatingActionButtonAndPopover.js

```diff
@@ -0,0 +1,252 @@
+import React from 'react';
+import _ from 'underscore';
+import {withOnyx} from 'react-native-onyx';
+import PropTypes from 'prop-types';
+import lodashGet from 'lodash/get';
+import {View} from 'react-native';
+import styles from '../../../../styles/styles';
+import * as Expensicons from '../../../../components/Icon/Expensicons';
+import Navigation from '../../../../libs/Navigation/Navigation';
+import ROUTES from '../../../../ROUTES';
+import Permissions from '../../../../libs/Permissions';
+import * as Policy from '../../../../libs/actions/Policy';
+import PopoverMenu from '../../../../components/PopoverMenu';
+import CONST from '../../../../CONST';
+import FloatingActionButton from '../../../../components/FloatingActionButton';
+import compose from '../../../../libs/compose';
+import withLocalize, {withLocalizePropTypes} from '../../../../components/withLocalize';
+import withWindowDimensions from '../../../../components/withWindowDimensions';
+import ONYXKEYS from '../../../../ONYXKEYS';
+import withNavigation from '../../../../components/withNavigation';
+import * as Welcome from '../../../../libs/actions/Welcome';
+import withNavigationFocus from '../../../../components/withNavigationFocus';
+import withDrawerState from '../../../../components/withDrawerState';
+
+/**
+ * @param {Object} [policy]
+ * @returns {Object|undefined}
+ */
+const policySelector = policy => policy && ({
+    type: policy.type,
+    role: policy.role,
+});
+
+const propTypes = {
+    /* Callback function when the menu is shown */
+    onShowCreateMenu: PropTypes.func,
+
+    /* Callback function before the menu is hidden */
+    onHideCreateMenu: PropTypes.func,
+
+    /** The list of policies the user has access to. */
+    allPolicies: PropTypes.shape({
+        /** The policy name */
+        name: PropTypes.string,
+    }),
+
+    /* Beta features list */
+    betas: PropTypes.arrayOf(PropTypes.string),
+
+    ...withLocalizePropTypes,
+};
+const defaultProps = {
+    onHideCreateMenu: () => {},
+    onShowCreateMenu: () => {},
+    allPolicies: {},
+    betas: [],
+};
+
+/**
+ * Responsible for rendering the {@link PopoverMenu}, and the accompanying
+ * FAB that can open or close the menu.
+ */
+class FloatingActionButtonAndPopover extends React.Component {
+    constructor(props) {
+        super(props);
+
+        this.showCreateMenu = this.showCreateMenu.bind(this);
+        this.hideCreateMenu = this.hideCreateMenu.bind(this);
+
+        this.state = {
+            isCreateMenuActive: false,
+        };
+    }
+
+    componentDidMount() {
+        const routes = lodashGet(this.props.navigation.getState(), 'routes', []);
+        Welcome.show({routes, showCreateMenu: this.showCreateMenu});
```

Comment:
✋ Coming from https://github.com/Expensify/App/issues/20609

The FAB button shows up on the Abracadabra page after opening the main browser tab and going back to the Abracadabra page.

For the fix, we conditionally show the popover only on the `CENTRAL_PANE_NAVIGATOR` or the home page.